examples/chaiscript/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-ChaiScript)
+
+# DOCUMENTATION_START {
+hunter_add_package(chaiscript)
+find_package(chaiscript CONFIG REQUIRED)
+# DOCUMENTATION_END }
+
+if (CMAKE_HOST_UNIX) 
+    #set unix link libraries
+    list(APPEND LIBS "dl") 
```

Comment:
There is no explicit usage of `dl` or `pthread` in `examples/chaiscript/main.cpp` I guess it means it's a chaiscript dependency. If so then it should be resolved in `chaiscriptConfig.cmake` file. See for example:
* https://github.com/elucideye/drishti/blob/a7f42af6bf1c6e367ff400b79c27c96967d72886/cmake/Config.cmake.in#L38-L55

Reply:
will correct.